fun/envs/common.py

```diff
@@ -449,6 +449,31 @@ def configure_haystack(elasticsearch_conf):
             'launch_url': 'https://marsha\.education/lti/videos/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}'
         },
     },
+    {
+        'display_name': 'Glowbl',
+        'is_launch_url_regex': False,
+        'hidden_fields': ['accept_grades_past_due', 'ask_to_send_email', 'custom_parameters', 'has_score', 'hide_launch', 'modal_height', 'modal_width', 'weight'],
+        'automatic_resizing': True,
+        'inline_ratio': 0.5625,
+        'defaults': {
+            'ask_to_send_username': True,
+            'description': """<img src='https://glowbl.com/wp-content/uploads/2017/04/Logo2.png'>
+                        <h1>Accès à la salle de visioconférence Glowbl</h1>
+                        <p>
+                        En cliquant sur ce bouton, vous quittez la plateforme <b>fun-corporate.fr</b>
```

Comment:
It seems to be fun-corporate-driven. Are we using this configuration for a single customer?

Reply:
👀 👍